en/docs/Learn/DesignAPI/CreateAPI/create-a-rest-api.md

```diff
@@ -31,18 +31,25 @@ Follow the instruction below to create a REST API.
 4.  When click **CREATE** or **CREATE & PUBLISH**, page will be redirected to the overview page of the newly created API. [![](../../../assets/img/Learn/overviewpage-rest-api.jpg)](../../../assets/img/Learn/overviewpage-rest-api.jpg)
 
 ## Design Configuration
-  Navigate to **Design Configurations** page and add tag as **pizza**.
+  1. Navigate to **Design Configurations** page.
+
+     <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>By default **Publisher Access Control** allowed for **All** users who has creator permission (see more <a href='../../AdvancedTopics/enable-publisher-access-control-in-api-publisher-portal'>Publisher Access Control</a> ) and **Developer Portal Visibility** available in **Public** (see more <a href='../../AdvancedTopics/control-api-visibility-and-subscription-availability-in-developer-portal/'> 
+      Developer Portal visibility </a>
+      )</p>
+      <p>
+      </div>
+    </html>
+
+   2. Add tag as **pizza**.
         <html><div class="admonition info">
         <p class="admonition-title">Info</p>
         <p>Tags can be used to filter out APIs matching certain search criteria. It is recommended that you add tags that explain the functionality and purpose of the API as subscribers can search for APIs based on the tags.</p>
         </div>
         </html>
 
-  <html><div class="admonition note">
-      <p class="admonition-title">Note</p>
-      <p>By default **Publisher Access control** allowed for **All** and **Developer Portal visibility** available for **Public**.</p>
-      </div>
-    </html>
+   **Make this the Default Version** checkbox ensures that the API is available in the Gateway without a version specified in the production and sandbox URLs.  This option allows you to create a new version of an API and set it as the default version. Then, the same resources can be invoked in the client applications without changing the API gateway URL. This allows you to create new versions of an API with changes, while at the same time allowing existing client applications to be invoked without the client having to change the URLs.
```

Comment:
Shall we change this to, **Make this the Default Version**  option allows you to set the created APIs version as the default version. The default version of the API can be invoked without specifying a version in the URL. This allows you to invoke resources in the client applications without changing the API gateway URL. this also allows you to create new versions of an API with changes, while at the same time allowing existing client applications to be invoked without the client having to change the URLs.